### DISCUSSION TITLE
Add support for filenames without file ending in 'Open file from URL'

### DISCUSSION THREAD

**Discussion by dbruggner** [foxglove/community#469] [Q&A]
Currently you can only open bag and mcap files with a file ending in 'Open file from URL' dialog. There are also cases where you have an auto-generated URL from some webspace without file ending which you want to use as the URL.
It would be helpful if you could support filename without file endings, a user could specify the file type by a checkbox.

**Comment by foxymiles** [comment]
Also reported in https://github.com/foxglove/studio/issues/4048 cc @tgoelles.
